extensions/panache/rest-data-panache/runtime/src/main/java/io/quarkus/rest/data/panache/ResourceProperties.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.rest.data.panache;
+
+import static java.lang.annotation.ElementType.TYPE;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+@Documented
+@Retention(RUNTIME)
+@Target({ TYPE })
+public @interface ResourceProperties {
+
+    /**
+     * Generate operations that support HAL content type.
+     * HAL methods are generated in addition to the standard methods. They accept the same parameters but return a content of
+     * `application/hal+json` type.
+     * The operations that support HAL responses are `get`, `list`, `add` and `update`.
+     *
+     * Default: false.
+     */
+    boolean hal() default false;
+
+    /**
+     * URL path segment that should be used to access the resources.
```

Comment:
I assume this is used in addition to whatever the Class uses, right?

Reply:
Looking at the tests, I see it does :)

Reply:
Yes, I've added a `@Path` annotation to the class too

Reply:
Probably worth mentioning in the javadoc both here and on the `OperationsProperties.path` one to explain they accumulate.